### ISSUE TITLE
Does this package cause a great breaking change to the " odavid/typeorm-transactional-cls-hooked"

### ISSUE THREAD

**Issue by Caesarsage** [Aliheym/typeorm-transactional#30]
As the package https://github.com/odavid/typeorm-transactional-cls-hooked does not support transaction for new nestJS version , how much changes will this affect my codebase aready written in https://github.com/odavid/typeorm-transactional-cls-hooked ?

**Comment by sunjoong85** [comment]
odavid/typeorm-transactional-cls-hooked dose not work on typescript version 3.x.x.

**Comment by Caesarsage** [comment]
@sunjoong85 yeah i am aware of that, I am asking if rewriting my codebase with Aliheym /
[typeorm-transactional](https://github.com/Aliheym/typeorm-transactional) will not cause much breaking point ?
